src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="card" data-qa="card">
+    <img
+      class="card__photo"
+      src="./images/imac.jpeg"
+      alt="card-imac">
+    <h2 class="card__name">
+      APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+    </h2>
+    <p class="card__code">
+      Product code: 195434
+    </p>
+    <div class="card__star_reviews">
+      <div class="stars">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <p class="card__reviews">
+        Reviews: 5
+      </p>
+    </div>
+    <div class="card__cost">
```

Comment:
```suggestion
    </div>
    
    <div class="card__cost">
```
It would be great to provide newlines between blocks of code

Reply:
Shall I use newlines between semantic blocks as well (block img-newline, block h2-newline, block  p-newline etc)?

Reply:
It depends on situation. Here is enough to separate as you did